### PR TITLE
feat: redesigning home page

### PR DESCRIPTION
**Pull request by BinaHovav**
This will be an ongoing PR for designing the new homepage.

### CODE REVIEW

File: src/pages/historicTimeline/index.tsx [hunk 1]
```diff
@@ -140,11 +140,15 @@ const TimelinePage = () => {
...
+      <Title className="page-title" level={3}>
+        הסטוריית נסיעות
+      </Title>
+      <Alert message="רשימת זמני עצירה בתחנה שנבחרה" type="info" />
       <Title level={3}>{t('rides_history')}</Title>
       <Space direction="vertical" size="middle" style={{ marginBottom: '22px' }}>
         <Alert message={t('timestamp_list')} type="info" />
       </Space>
-
+      main
```
Comment (NoamGaash) [inline src/pages/historicTimeline/index.tsx:151]:
```suggestion
```

File: src/routes/index.tsx [hunk 2]
```diff
@@ -89,7 +97,7 @@ export const PAGES = [
 
 const getRoutesList = () => {
   const pages = PAGES
-  const RedirectToDashboard = () => <Navigate to={pages[0].path} replace />
+  const RedirectToHome = () => <Navigate to={pages[0].path} replace />
```
Comment (NoamGaash) [inline src/routes/index.tsx:100]: let's do this change after the page is ready

File: src/routes/index.tsx [hunk 3]
```diff
@@ -110,8 +118,10 @@ const getRoutesList = () => {
         }}
       />
       <Route path="data-research" element={<DataResearch />} />
+      <Route path="*" element={<RedirectToHome />} key="back" />
       <Route path="release" element={<iframe src="https://noam-gaash.co.il/databus/" />} />
-      <Route path="*" element={<RedirectToDashboard />} key="back" />
+      <Route path="*" element={<RedirectToHome />} key="back" />
+      main
```
Comment (NoamGaash) [inline src/routes/index.tsx:124]:
```suggestion
```

**Review by NoamGaash — COMMENTED** [review]
great start!
but let's postpone the navbar changes until the page is ready
see here - https://github.com/hasadna/open-bus-map-search/pull/388
how I hide pages from the menu

File: src/routes/index.tsx (hunk 2 above)
Comment (BinaHovav) [reply src/routes/index.tsx:100]: good idea

File: src/pages/homepage/HomePage.scss
Comment (NoamGaash) [inline src/pages/homepage/HomePage.scss:1]: an this file be removed?

File: tests/dashboard.spec.ts [hunk 4]
```diff
@@ -21,6 +21,8 @@ test('dashboard is on homepage', async ({ page, advancedRouteFromHAR }) => {
   })
   await page.goto('/')
   await page.getByText('הקווים הגרועים ביותר').click()
+  // await page.waitForSelector('h1:has-text("This is soon going to be a home page")')
+
```
Comment (NoamGaash) [inline tests/dashboard.spec.ts:25]: can be removed (?)

**Review by NoamGaash — COMMENTED** [review]
you're doing great, thanks for merging with main

File: src/pages/homepage/HomePage.scss
Comment (BinaHovav) [reply src/pages/homepage/HomePage.scss:1]: why removed? i need it for the styles

File: tests/dashboard.spec.ts (hunk 4 above)
Comment (BinaHovav) [reply tests/dashboard.spec.ts:25]: sure

File: src/pages/homepage/HomePage.scss
Comment (NoamGaash) [reply src/pages/homepage/HomePage.scss:1]: it's an empty file

Comment (BinaHovav) [reply src/pages/homepage/HomePage.scss:1]: not for long :)

**Comment by BinaHovav** [comment]
I'm closing this PR and opening a cleaner one.

File: src/img/busImg.png
Comment (NoamGaash) [inline src/img/busImg.png:1]: if we're going to use that, please add an attribution section in bottom of the about page

**Pull request closed without merging**